website/src/documentation/sdks/nexmark.md

```diff
@@ -347,6 +355,13 @@ These tables contain statuses of the queries runs in the different runners. Goog
       <td>ok</td>
       <td>ok</td>
     </tr>
+    <tr>
+      <td>BOUNDED_SIDE_INPUT_JOIN</td>
```

Comment:
BTW from my reading of the bug the problem with Dataflow runner is not this query. So it is "OK" for all just like you have here.

Reply:
Good do know !